src/wrapped/wrappedsdl2.c

```diff
@@ -681,6 +782,67 @@ EXPORT void my2_SDL_Log(x86emu_t* emu, void* fmt, void *b) {
     #endif
 }
 
+EXPORT int my2_SDL_GL_SetAttribute(x86emu_t* emu, uint32_t attr, int value) 
+{
+    sdl2_my_t *my = (sdl2_my_t *)emu->context->sdl2lib->priv.w.p2;    
+    return my->SDL_GL_SetAttribute(attr, value);
+}
+
+// Cut down and adapted from SDL_video.c
+EXPORT int my2_SDL_GL_ExtensionSupported(x86emu_t* emu, char *name) 
```

Comment:
Maybe this one should be inside an `#ifdef GOA_CLONE`?

Reply:
Likely makes more sense to check for gl4es than GOA_CLONE imho, since that's the reason for this functionality.